cli/list-pause-plans.js

```diff
@@ -0,0 +1,73 @@
+import { ethers } from 'ethers';
+import { table } from 'table';
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
+import pauseABI from './pause_abi.json' with { type: 'json' };
+import { getFilteredEvents, prepareData } from "./pause-plan-data.js";
+
+const argv = yargs(hideBin(process.argv))
+    .option('pending', {
+        alias: 'p',
+        type: 'boolean',
+        description: 'Show only pending plans',
+        default: false
+    })
+    .option('from-block', {
+        alias: 'b',
+        type: 'number',
+        description: 'Display spells from a given block',
+        default: 0
+    })
+    .option('eth-rpc-url', {
+        alias: 'e',
```

Comment:
Maybe it would be better to make the options align with Foundry:

![image](https://github.com/user-attachments/assets/462f5ffe-f2e6-48db-87f4-09b5f16d413e)

I agree can be f'ing confusing, but ¯\\\_(ツ)_/¯:

```suggestion
    .option('rpc-url', {
        alias: ['fork-url', 'f'],
```